src/middlewares/verifyUser.js

```diff
@@ -0,0 +1,36 @@
+import verifyToken from '../utils/verifyToken';
+import db from '../models';
+
+const verifyUser = async (req, res, next) => {
+  try {
+    const { token } = req.query;
+    const userInfo = verifyToken(token);
+    const user = await db.User.findOne({
+      where: {
+        id: userInfo.id
+      }
+    });
+    if (!user) {
+      return res.status(403).json({
+        status: 403,
+        message: res.__('you are not allowed to perform this action'),
+      });
+    }
+
+    req.user = user;
+    return next();
+  } catch (err) {
+    if (err.name === 'JsonWebTokenError') {
+      return res.status(400).json({
+        status: 400,
+        message: res.__('token must be provided and valid'),
+      });
+    }
+    return res.status(500).json({
```

Comment:
could you use responseHandler.js to return the response?

Reply:
done